dnf/plugins/repoquery/dnf-command-repoquery.c

```diff
@@ -56,6 +58,43 @@ gptrarr_dnf_package_cmp (gconstpointer a, gconstpointer b)
   return dnf_package_cmp(*(DnfPackage**)a, *(DnfPackage**)b);
 }
 
+static void
+package_info_add_line (struct libscols_table *table, const char *key, const char *value)
+{
+  struct libscols_line *ln = scols_table_new_line (table, NULL);
+  scols_line_set_data (ln, 0, key);
+  scols_line_set_data (ln, 1, value);
+}
+
+static void
+print_package_info (DnfPackage *package)
+{
+  struct libscols_table *table = scols_new_table ();
+  scols_table_enable_noheadings (table, 1);
+  scols_table_set_column_separator (table, " : ");
+  scols_table_new_column (table, "key", 5, 0);
+  struct libscols_column *cl = scols_table_new_column (table, "value", 10, SCOLS_FL_WRAP);
+  scols_column_set_safechars (cl, "\n");
+  scols_column_set_wrapfunc (cl, scols_wrapnl_chunksize, scols_wrapnl_nextchunk, NULL);
+
+  package_info_add_line (table, "Name", dnf_package_get_name (package));
+  package_info_add_line (table, "Version", dnf_package_get_version (package));
+  package_info_add_line (table, "Release", dnf_package_get_release (package));
+  package_info_add_line (table, "Architecture", dnf_package_get_arch (package));
```

Comment:
Are we missing the Epoch here? `dnf repoquery --info` prints the `Epoch : ` line only if the epoch is specified for a package... (I haven't gone checking if we're missing more optional fields here...)

Reply:
Thanks. I added `Epoch`. It is printed only if epoch != 0. Similar to dnf.
Another missing is `From repo`. This will be out of scope now.